src/dashboard/src/components/api/views.py

```diff
@@ -655,10 +660,8 @@ def path_metadata(request):
     path = request.GET.get('path', '') if request.method == 'GET' else request.POST.get('path', '')
 
     # Get current metadata, if any
-    file_lod = get_object_or_None(models.SIPArrange, arrange_path=path, sip_created=False)
-    if file_lod is None:
-        # Try with trailing / to see if it's a directory
-        file_lod = get_object_or_None(models.SIPArrange, arrange_path=path + '/', sip_created=False)
+    file_lod = models.SIPArrange.objects \
+        .get(arrange_path__in=(path, path + '/'), sip_created=False)
```

Comment:
Shouldn't there still be a `try/except` here? I mean, can't there still be an unmatching `SIPArrange`? Also, is it possible for there to be matches for both `arrange_path`s and for your new code to return the `/`-suffixed one where the original code would have returned the `/`-less one?